packages/amplify-category-api/src/provider-utils/awscloudformation/utils/edit-schema-flow.ts

```diff
@@ -8,7 +8,7 @@ export const editSchemaFlow = async (context: any, apiName: string) => {
     type: 'confirm',
     name: 'editNow',
     message: 'Do you want to edit the schema now?',
-    default: false,
+    default: true,
```

Comment:
was this supposed to be switched?

Reply:
Yes, this is intentional. We are changing the default option to 'Y' for this prompt.